frontend/preview/spec.html

```diff
@@ -0,0 +1,1549 @@
+        index an array out of bounds, accessing a map value for a key that does not exist, or a
+        failed type assertion. When a run-time panic occurs, the Evy program will stop and error
+        details will be printed. You can trigger a panic in your own code by calling the built-in
+        function <code>panic &quot;msg&quot;</code>.
+      </p>
+      <p>
+        <strong>Recoverable errors</strong> are errors that can be handled by the Evy program. They
+        are typically caused by user input or external factors that the Evy program cannot control.
+        Functions that can cause recoverable errors set the global <code>err</code> variable to
+        <code>true</code> and the string variable <code>errmsg</code> to a description of the error.
+        The Evy program can then check the value of <code>err</code> and handle the error
+        accordingly. You can trigger a recoverable error in your own code by setting
+        <code>err</code> and <code>errmsg</code>.
+      </p>
+      <p>
+        For more information on run-time panics and recoverable errors, see the built-in
+        documentation on the <a href="builtins.html#panic">panic function</a> and the
+        <a href="builtins.html#errors">errors section</a>.
+      </p>
+      <h2>Execution Model and Event Handlers</h2>
+      <p>
+        Evy first executes all top-level code in the order it appears in the source code. If there
+        is at least one event handler, Evy then enters an event loop. In the event loop, Evy waits
+        for external events, such as a key press or a pointer down event. When an event occurs, Evy
+        calls the corresponding event handler function if it has been implemented. The event handler
+        function can optionally receive arguments, such as the key character or the pointer
+        coordinates. Once the event handler function has finished, Evy returns to the event loop and
+        waits for the next event.
+      </p>
+      <p>
+        Event handlers are declared using the <code>on</code> keyword. Only predefined events can be
+        handled: <code>key</code>, <code>down</code>, <code>up</code>, <code>move</code>,
+        <code>animate</code>, and <code>input</code>. The parameters to the event handlers must
+        match the expected signature. The parameters can be fully omitted or fully specified. If
+        only some parameters are needed, use the anonymous <code>_</code> parameter.
+      </p>
+      <p>
+        For more information on individual event handlers, see the
+        <a href="builtins.html#event-handlers">built-in documentation</a>.
+      </p>
+      <h2>Runtimes</h2>
+      <p>
+        Evy has two runtimes: the <strong>terminal runtime</strong> and the
+        <strong>browser runtime</strong>.
+      </p>
+      <p>
+        The browser runtime can be tried at [play.evy.dev]. It fully supports all built-in functions
```

Comment:
The `[play.evy.dev]` didn't linkify. I wonder why. The other reference-style links did linkifiy - I wonder if it's because of the url-looking bit or if its because the link is the last thing in the file and there's some off-by-1 thing somewhere?

Reply:
maybe cause the https:// is missing? can't believe you noticed this....

Reply:
oh. nvm.

Reply:
Fixed, this was a mistake in spec.md